contributions/README.md

```diff
@@ -0,0 +1,79 @@
+# Users
+
+## Contributions Object
+
+```
+{
+  prList: [{
+    'prTitle': <title of pull request>,
+    'prUrl': <url of pull request>,
+    'state': <open or closed>,
+    'prCreatedAt': <unix_timestamp>,
+    'prUpdatedAt': <unix_timestamp>,
+    'prRaisedBy': <username of the assignee>
```

Comment:
Do we need to have a "pr" prefix in the keys? The 'pr' prefix feels redundant. We know it's a PR object, so having simple `title`, `URL`, `createdAt`, etc should suffice

Reply:
Ok Makes sense, I have updated the object accordingly 9ad7b536cc2fa4ca074c4a616844d1c2e5389ffc